src/lib/y2users/user.rb

```diff
@@ -213,8 +213,9 @@ def password_issues
     # @return [Boolean]
     def system_uid?
       return false unless uid && sys_uid_max
+      return false if uid == ""
 
-      uid.between?(sys_uid_min || 1, sys_uid_max)
+      uid.to_i.between?(sys_uid_min || 1, sys_uid_max)
```

Comment:
I already face this confusion few times, maybe it is time to reconsider having uid as string  and not Integer